tests/reporter/kubernetes.go

```diff
@@ -244,6 +245,30 @@ func (r *KubernetesReporter) logVMIs(virtCli kubecli.KubevirtClient, vmis *v12.V
 	fmt.Fprintln(f, string(j))
 }
 
+func (r *KubernetesReporter) logDVs(virtCli kubecli.KubevirtClient) {
```

Comment:
This API might not be available on all test clusters. It might be good to check this to reduce noise. WDYT?

Reply:
good idea, DV is not available if CDI is not deployed, 
@xpivarc do you know the simplest way to check it?

Reply:
It should be possible with client and `DiscoveryInterface`

Reply:
will try

Reply:
@xpivarc  done, wdyt?

Reply:
We could use `sync.Once` to optimize since the availability should not change.

Reply:
Yes, we could, but it is pretty fast call (only gets registered resource types), it is only run on failures, currently at most 10 reports are done. 

I would need to add a flag to KubernetesReporter like `KubernetesReporter.dvAvailable`, then use it here. You need to convince me it is worth doing ;)

Reply:
I think this is valuable for debugging as is so I will not try to convince you.